pyproject.toml

```diff
@@ -127,7 +127,10 @@ dependencies = [
   "types-requests",
   "types-protobuf",
   "openai>=1.0.0",
-  "litellm>=1.0.3"
+  "litellm>=1.0.3",
+  "tornado",
+  "websockets",
+  "rich",
```

Comment:
Added these because I ran into these errors with mypy.
```
...
AssertionError: Cannot find module for tornado.concurrent.Future
...
AssertionError: Cannot find module for websockets.legacy.server.WebSocketServerProtocol
...
AssertionError: Cannot find module for rich
```